src/pointer/exception/Methods.java

```diff
@@ -0,0 +1,50 @@
+package pointer.exception;
+
+public class Methods {
+
+    private Methods() {}
+
+    public static int sum(int a, int b) throws MyException, IllegalAccessException, MyException2 {
+        verifyParameters(a, b);
+        return a + b;
+    }
+
+    public static int subtract(int a, int b) throws MyException, IllegalAccessException, MyException2 {
+        verifyParameters(a, b);
+        return a - b;
+    }
+
+    public static int multiply(int a, int b) throws MyException, IllegalAccessException, MyException2 {
+        verifyParameters(a, b);
+        return a * b;
+    }
+
+    public static int divide(int a, int b) throws MyException, IllegalAccessException, MyException2 {
+        verifyParameters(a, b);
+        return a/b;
+    }
+
+    private static void verifyParameters(int a, int b) throws IllegalAccessException, MyException, MyException2 {
+        if (a < 0 && b < 0)
```

Comment:
if{
}

Reply:
And for single operator?
if (condition){
    doSomething();
}